starter-code/src/App.js

```diff
@@ -1,21 +1,72 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import foods from "./foods.json";
+import "./App.css";
+import "bulma/css/bulma.css";
+import AddFood from "./components/AddFood";
+import Search from "./components/Search";
+import FoodList from "./components/FoodList";
+import TodayFoods from "./components/TodayFoods";
 
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      foodData: foods,
+      showForm: false,
+      todayFood: []
+    };
+  }
+
+  addFood = item => {
+    const foodArray = [...this.state.foodData];
+    foodArray.push(item);
+    this.setState({
+      foodData: foodArray
+    });
+  };
+
+  formVisibility = () =>
+    this.setState({
+      showForm: !this.state.showForm
+    });
+
+  searchFood = search => {
+    const results = [];
+    const foodList = [...this.state.foodData];
+    foodList.map(f => {
+      return f.name.toLowerCase().includes(search.toLowerCase())
+        ? results.push(f)
+        : " ";
+    });
```

Comment:
You are abusing map as a loop here. I would either use a loop/foreach or use the filter method:

```js
const results = [];
this.state.foodData.forEach(f => {
  if (f.name.toLowerCase().includes(search.toLowerCase())) {
    result.push(f);
  }
});
// or
const results = this.state.foodData.filter(
  f => f.name.toLowerCase().includes(search.toLowerCase())
);